mypyc/primitives/registry.py

```diff
@@ -354,6 +355,7 @@ def c_method_op(name: str,
                 var_arg_type: Optional[RType] = None,
                 truncated_type: Optional[RType] = None,
                 ordering: Optional[List[int]] = None,
+                extra_int_constant: Optional[int] = None,
```

Comment:
Include also the type of the integer. For non-C backends in particular it will be useful to know whether it's a 32-bit or 64-bit integer, for example. Sometimes we'll also want to pass null pointers, and it would be nice to be able to specify their types. A reasonable way would be to make this argument `Optional[Tuple[int, RType]]`.

Update docstring to document this.